doc/tutorials/starship.rst

```diff
@@ -37,46 +40,56 @@ To test connectivity, you can use the following command:
    You have successfully authenticated to the platform!
 
 
+.. _first_program:
+
 Executing your first program
 ============================
-The easiest way to execute a program using StarshipEngine is to create a blackbird script (an ``xbb`` file) and place it in your current working directory. Check the :ref:`blackbird tutorial <tutorial>` for how to create this file.
 
-For this example, we will use the following file and save it to ``test.xbb`` in our current working directory:
+The easiest way to execute a program using StarshipEngine is to create a Blackbird script (an ``xbb`` file)
+and place it in your current working directory. Check the :ref:`Blackbird tutorial <blackbird>` for how to create this file.
+
+For this example, consider the following Blackbird script, which represents a quantum program that matches
+exactly the gate layout of the Chip0 photonic hardware device. We will save the following file as ``test.xbb``
+in our current working directory:
 
 .. code-block:: python
 
-   name template_2x2_chip0
-   version 1.0
-   target chip0 (shots = 50)
+   name template_2x2_chip0     # Name of the program
+   version 1.0                 # Blackbird version number
+   target chip0 (shots = 50)   # This program will run on chip0 for 50 shots
 
-   float sqz0 = 1.0
-   float sqz1 = 1.0
+   # Define the interferometer phase values
    float phi0 = 0.574
    float phi1 = 1.33
 
-   # for n spatial degrees, first n signal modes, then n idler modes, phase zero
-   S2gate(sqz0, 0.0) | [0, 2]
-   S2gate(sqz1, 0.0) | [1, 3]
+   # Initial states are two-mode squeezed states,
+   # applied to alternating pairs of modes.
+   S2gate(1.0, 0.0) | [0, 2]
+   S2gate(1.0, 0.0) | [1, 3]
 
-   # standard 2x2 interferometer for the signal modes (the lower ones in frequency)
+   # A standard 2x2 interferometer is applied
+   # to the first pair of modes
    Rgate(phi0) | [0]
```

Comment:
Just the structure of the chip. The chip uses Mach-Zehnder interferometers, consisting of two parametrized phase-shifters, and two fixed 50-50 beamsplitters. Since the beamsplitters are not tunable, the transmissivity and phase are determined by the internal and external phase shifters.